_posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md

```diff
@@ -0,0 +1,115 @@
+		* Most venues require use of their in-house catering service. If the venue does not offer catering services, check whether the venue restricts catering access to an approved vendor list, and what, if any, requirements they have for outside catering vendors.
+		* Adjoins or within walking distance to a suitable hotel which can accommodate 350-450 attendees
+		* Excellent Internet connectivity
+		* [Accessible](https://modelviewculture.com/pieces/organizing-more-accessible-tech-events) to as many attendees as possible
+		* Reasonable access to transit for sightseeing
+		* Gender-neutral (or at least gender-inclusive) restrooms within a short walk to the conference rooms.
+
+	* The venue does not have to be a hotel or corporate conference center. Many universities offer conference services, and there are also municipal/county level event venues which can be likely sites for DjangoCon US.
+
+3. A short list of possible sprint venues, with the following information:
+
+	* Name
+	* Location
+	* Transportation options (how do people get there from the hotel?)
+	* Attendance capacity
+	* Cost of function space, catering, and audio-visual services. Sprints are often hosted by local companies as sponsorship of the event, which is preferable, as it saves the cost of renting a venue.
+	* A good sprint venue is/has the following:
+		* Easy to reach from the conference hotel
+		* Free parking if a car is required for access
+		* A room to hold 50-75 people, with work tables, chairs, power, and internet connectivity
+		* See catering requirements above. Eating tables should be separate from work tables if at all possible.
+		* Excellent Internet connectivity.
+
+4. Hotel. The ideal hotel is adjacent to the conference venue, or a very short walk away. For the hotel, we need:
+
+	* Name
+	* Location
+	* Transportation options (how do people get there from the airport?)
+	* The conference room rate (should be discounted from the standard rate)
+	* A block of rooms starting the two nights before tutorials, and running through the last night of the sprints. The breakdown of the room counts would be estimated at:
+	  * Friday - 10
+	  * Saturday - 15
+	  * Sunday Tutorials - 100
+	  * Monday Main Conf - 100
+	  * Tuesday Main Conf - 100
+	  * Wednesday Main Conf - 80
+	  * Thursday Sprints - 30
+	  * Friday Sprints - 15
+
+	* The block should include the guaranteed ability to add 20% more rooms for any given night if needed without moving to an overflow hotel.
+	* Concessions policy (comp room nights per actualized room nights)
+	* A sample contract showing the terms of booking the block of rooms. Again, do not sign any contracts. DEFNA will handle the business side of things. We just need to see the terms the hotel proposes for the group block.
+
+5. Provide estimates on costs associated with flights, accommodation, meals, ground transportation, etc. for an overview of total cost for attendees.
```

Comment:
Should we add examples here like costs for people flying in from major cities like LA, Chicago, NY, etc ?

Reply:
Sure, that sounds good to me.

Reply:
How about something like:

> Suggested cost examples include typical round-trip airfare from several major hub cities, including New York, Los Angeles, San Francisco, Atlanta, Dallas, Chicago, and London.

Reply:
Yes, looks good!